ocs_ci/deployment/deployment.py

```diff
@@ -199,6 +200,102 @@ def do_deploy_submariner(self):
             submariner = Submariner()
             submariner.deploy()
 
+    def do_gitops_deploy(self):
+        """
+        Deploy GitOps operator
+
+        Returns:
+
+        """
+        if not config.ENV_DATA.get("deploy_gitops_operator"):
+            return
+
+        # Multicluster operations
+        if config.multicluster:
+            config.switch_acm_ctx()
+            logger.info("Creating GitOps Operator Subscription")
+            gitops_subscription_yaml_data = templating.load_yaml(
+                constants.GITOPS_SUBSCRIPTION_YAML
+            )
+            package_manifest = PackageManifest(
+                resource_name=constants.GITOPS_OPERATOR_NAME,
+            )
+            gitops_subscription_yaml_data["spec"][
+                "startingCSV"
+            ] = package_manifest.get_current_csv(
+                channel="latest", csv_pattern=constants.GITOPS_OPERATOR_NAME
+            )
+
```

Comment:
If possible, pls add a few log messages which helps in understanding/debugging.

Reply:
i don't think we need to add any logs here it just build yaml